frameworks/AutoGluonTS/exec.py

```diff
@@ -88,28 +90,43 @@ def run(dataset, config):
                   target_is_encoded=False,
                   models_count=num_models_trained,
                   training_duration=training.duration,
-                  predict_duration=predict.duration)
+                  predict_duration=predict.duration,
+                  quantiles=predictions.iloc[:, 1:])
```

Comment:
can make more explicit by dropping the 'mean' column by name rather than by assumed position. This makes it easier to understand.